### PR TITLE
Update Pathology Tests

### PR DESCRIPTION
**Pull request by behxyz**
### Description
Implement `download_if_not_exist` in tests/utils.py and update all the pathology tests that need to download the WSI test case.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [x] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -582,5 +583,24 @@ def query_memory(n=2):
     return ",".join([f"{int(x)}" for x in ids])
 
 
+def download_if_not_exist(file_url, file_path=None):
```
Comment (wyli) [inline tests/utils.py:586]: could use https://github.com/Project-MONAI/MONAI/blob/master/monai/apps/utils.py#L74 we could have a hash for the file to detect external changes.  there's an example usage here: https://github.com/Project-MONAI/MONAI/blob/master/tests/clang_format_utils.py#L60

**Review by wyli — APPROVED** [review]
thanks for the quick fix!

**Comment by behxyz** [comment]
@wyli @Nic-Ma , 
I have seen PT18+CUDA112 is failing several times, is there any components that mainly causes this failure? 
https://github.com/Project-MONAI/MONAI/pull/1892/checks?check_run_id=2221781586#step:6:8088

**Comment by wyli** [comment]
yes conflictions from multiple jobs trying to use the same GPU devices, there's no proper scheduling here for the github actions

**Comment by behxyz** [comment]
@wyli @Nic-Ma, I see some housekeeping PRs are taking the resources while I have been waiting for more than 6 hours for pathology PRs to be merged and they are failing because of this. Can we hold on for a day on not urgent PRs?

**Pull request merged**